client/src/components/Recomendations/Recomendations.tsx

```
@@ -35,27 +37,45 @@ const Recomendations: React.FC = () => {
-                            ))}
+                            </div>
+                        ))}
                     </Marquee>
-                    
                 </ul>
             )}
-            {suggestions.length === 0 && !loading && <p>No suggestions available.</p>}
+            {suggestions.length === 0 && !loading &&
+                <>
+                    <p>No suggestion Available</p>
+                </>}
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)